### PR TITLE
Detach stats from sticky header on mobile

### PR DESCRIPTION
**Pull request by kyleshay**
Behind `$featureFlags.unstickyStats` on dev/beta, we now move the stats off of the sticky header on mobile only to provide some more space while scrolling. Desktop will remain unchanged.

When enabled and on a phone, the vault header will now show the capacity counts, because that may still be important to see while scrolling. Currencies will still be shown under the vault, but only when scrolled to the top.

I've considered moving some of the character stats to the character tile (so they were still visible while scrolling), but I didn't want to make additional design changes in this PR.

I wonder if it would be useful to have a "scroll to top" button on mobile? The view is quite long. Placement may be difficult (wouldn't want to cover any items.) Maybe in the header somewhere or next to search. 

I moved the majority of files that relate to both vault capacity, currencies, and character stats to a new directory `store-stats`

| View | Before | After |
| ---- | ---- | ---- |
| Desktop (identical) | Screen Shot 2020-08-11 at 3 22 32 PM | Screen Shot 2020-08-11 at 3 22 13 PM |
|Mobile (identical) | Screen Shot 2020-08-11 at 4 22 59 PM | Screen Shot 2020-08-11 at 4 22 44 PM |
| Mobile (scrolled) | Screen Shot 2020-08-11 at 4 08 41 PM | Screen Shot 2020-08-11 at 4 08 29 PM |

### CODE REVIEW

**Comment by bhollis** [comment]
At least on iOS, tapping the status bar always scrolls to the top in any app. 

I'm super in favor of this change, but last time we did it there was a lot of outrage. Maybe getting max power into the tile would help.

**Comment by kyleshay** [comment]
| no scroll | scroll |
| --- | --- | 
|Screen Shot 2020-08-12 at 1 07 10 PM|Screen Shot 2020-08-12 at 1 07 15 PM|

**Comment by kyleshay** [comment]
or this, Screen Shot 2020-08-12 at 1 11 17 PM

File: src/app/character-tile/CharacterTile.tsx [hunk 1]
```diff
@@ -11,11 +14,26 @@ const CharacterEmblem = ({ store }: { store: DimStore }) => (
...
+  return store.isVault;
+}
+
+interface StoreProps {
+  isPhonePortrait: boolean;
+}
+
+function mapStateToProps(state: RootState): StoreProps {
+  return {
+    isPhonePortrait: state.shell.isPhonePortrait,
```
[the remark this answers is not in the document]
Comment (kyleshay) [reply src/app/character-tile/CharacterTile.tsx:27]: ah, nice!

[the remark this answers is not in the document]
Comment (kyleshay) [reply src/app/character-tile/CharacterTile.tsx:27]: had an issues with that, and error message didn't seem super helpful at first glance. looks like it's working as expected in other files... reverted it back and will cleanup after.

https://travis-ci.com/github/DestinyItemManager/DIM/builds/179532562

**Pull request merged**